packages/common/src/templates/configuration/sandbox/ui.tsx

```diff
@@ -144,6 +144,20 @@ export const ConfigWizard = (props: ConfigurationUIProps) => {
           Which template to use for this sandbox.
         </ConfigDescription>
       </PaddedConfig>
+      {!currentTemplate.isServer ? (
+        <PaddedConfig>
+          <ConfigItem>
+            <PaddedPreference
+              title="Disable Console"
+              type="boolean"
+              {...bindValue(parsedFile, 'disableLogging')}
+            />
+          </ConfigItem>
+          <ConfigDescription>
+            Disable the in browser console to prevent slowing down of the page
```

Comment:
Oh wait, one thing, this one is tricky. I have this suggestion, because in 99% of the cases it doesn't slow down.

```suggestion
            Disable the in-browser console to prevent slowing down of the page when there are many logs to the console.
```